service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/NotificationHandler.java

```diff
@@ -153,7 +155,7 @@ public void sendNotificationsForIssuedCaseManagementOrder(CMOEvent event) {
         String localAuthorityCode = (String) caseDetails.getData().get(CASE_LOCAL_AUTHORITY_PROPERTY_NAME);
 
         sendCMONotificationForLocalAuthority(caseDetails, localAuthorityCode);
-        // TODO: 10/12/2019 FPLA-27 will add document related notifications
+        sendCMODocumentLinkNotificationForCafcass(caseDetails, localAuthorityCode, event.getDocument());
```

Comment:
Story says 'If party is to be served via email, send notification 1'. Where is condition checking what way party should be served?

Reply:
this is correct. I have added clarifying todos as it will be covered once 911 implemented